.github/workflows/gosec.yml

```diff
@@ -17,4 +17,4 @@ jobs:
       - name: Run Gosec Security Scanner
         uses: securego/gosec@master
         with:
-          args: --exclude-dir=kubetest-plugins --exclude-dir generatebundlefile --exclude-dir ecrtokenrefresher  ./...
```

Comment:
Why do we have this?

Reply:
This is for the github workflows

Reply:
Right, but why is gosec ignoring parts of our repo?

Reply:
Created issue #870 to follow up on this